### PR TITLE
Pin to last working version of bqplot-image-gl

### LINKED ISSUE #1309: Unpin bqplot-image-gl
Their 1.4.8 release broke so we temporarily pinned to 1.4.7. This is to unpin that when the newest release is fixed.

### PR DESCRIPTION
**Pull request by rosteen**
The 1.4.8 release of `bqplot-image-gl` is broken, let's not use it for now 😅 

Close #1309

### CODE REVIEW

**Comment by rosteen** [comment]
Followup here https://github.com/spacetelescope/jdaviz/issues/1309

**Review by javerbukh — APPROVED** [review]

**Review by duytnguyendtn — APPROVED** [review]
Thanks for the catch!

File: setup.cfg [hunk 1]
```diff
@@ -22,7 +22,7 @@ install_requires =
     matplotlib
     traitlets>=5.0.5
     bqplot>=0.12.33
-    bqplot-image-gl>=1.4.7
+    bqplot-image-gl==1.4.7
```
Comment (pllim) [inline setup.cfg:25]: Maybe more future-proof like this?
```suggestion
    bqplot-image-gl>=1.4.7,!=1.4.8
```

Comment (pllim) [reply setup.cfg:25]: I'll let @rosteen decide...

Comment (javerbukh) [reply setup.cfg:25]: He stepped out, is there any reason not to commit that?

Comment (pllim) [reply setup.cfg]: OK, let's YOLO and see if CI passes.

**Pull request merged**